travis/shared_configs/python-tester-conda-latest.yml

```diff
@@ -53,3 +52,5 @@ jobs:
                   python -m pytest
               fi
           fi
+      after_failure:
+        - cat logs/run_tests_log.txt
```

Comment:
This comes from our per-repo settings (e.g.)
https://github.com/pcdshub/typhos/blob/826db3b5778ae2f1e51d7c16a10c4ffbb8596a28/run_tests.py#L25

Would it be advisable to check for the existence of the file first, just in case not all repos have it? We had talked about removing these at some point, even, if I recall correctly...

Reply:
`happi` doesn't seem to have this log file, for example.

Reply:
If the file doesn't exist, we can't do much worse than having already failed

Reply:
Fair - just seems like unnecessary additional error. Something we might forget about in the future where it came from, even.

Reply:
Should we add this in other repos that don't have them then?

Reply:
Maybe the most proper thing is to make this configurable?

Reply:
`cat "${AFTER_FAILURE_LOGFILE}"`

Reply:
Configurable seems good to me, if that's preferable to you 👍

Reply:
Added small configurability and a file check with a more helpful message if there isn't a logfile